pkgs/tools/security/rhash/dont-fail-ln.patch

```diff
@@ -0,0 +1,8 @@
+--- a/librhash/Makefile
++++ b/librhash/Makefile
+@@ -32,1 +32,1 @@
+-	  ln -s $(LIBRHASH_SHARED) $(LIBDIR)/$(LIBRHASH_SO_MAJ)
++	  ln -s $(LIBRHASH_SHARED) $(LIBDIR)/$(LIBRHASH_SO_MAJ) || true
+@@ -180,1 +180,1 @@
+-	  ln -s $(LIBRHASH_SHARED) $(LIBRHASH_SO_MAJ)
++	  ln -s $(LIBRHASH_SHARED) $(LIBRHASH_SO_MAJ) || true
```

Comment:
does freebsd not support ln?

Reply:
This is a hackfix for an issue that will be fixed in the next version of rhash. I believe this is the relevant issue: https://github.com/rhash/RHash/issues/246